lib/services/service-proxy.ts

```diff
@@ -10,6 +10,7 @@ export class ServiceProxy implements CloudService.IServiceProxy {
 		path = `appbuilder/${path}`;
 		headers = headers || Object.create(null);
 		headers["X-Icenium-SolutionSpace"] = headers["X-Icenium-SolutionSpace"] || "Private_Build_Folder";
+		headers["User-Agent"] = `fusion/${require("../../package.json").version} (Node.js ${process.versions.node}; ${process.platform}; ${process.arch})`;
```

Comment:
for version you can reuse the code from here https://github.com/NativeScript/nativescript-cloud/blob/master/lib/commands/cloud-lib-version.ts#L11-L15